KalturaWowzaServer/release/register_media_server.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+require_once('../app/clients/php5/KalturaClient.php');
```

Comment:
You can't use this path, it's not necessarily installed or of the right version, have your own client library as part of your project.